haskell/src/Ligo/BaseDAO/Types.hs

```diff
@@ -779,7 +782,7 @@ mkConfig customEps votingPeriod fixedProposalFee maxChangePercent changePercent
   }
 
 defaultConfig :: Config
-defaultConfig = mkConfig [] 10 0 19 5 500
+defaultConfig = mkConfig [] (Period 20) (FixedFee 0) 19 5 (GovernanceTotalSupply 500)
```

Comment:
This is how I got the CI tests to pass. I am not sure if it makes sense, but I doubled the period duration because the block time was halfed post granada protocol. So I thought may be it is causing the timing overflows, and it appear to have worked, at least for the timebeing...

Reply:
I keep getting the tests fail in localchain too, not sure why. The rests look good to me.

Reply:
AFAIR the problem we had before with the period time was that, if blocks are not processed very quickly, then when we wait before making a new transfer too much time might pass by (because we waited for much of it in the first call).

It seems that local-chain on hangzhounet is suffering from these delays (and had some more trouble a few days back too) and it appears to me that we are now getting these consistently.

AFAIK (please check/correct me if I'm wrong), then all that this change is that tests will take longer, which is not too bad. So IMO we can get this merged and keep an eye on the local-chain efficiency.

Reply:
It looks like some tests on local-chain still failed on the last run :disappointed: 

I just restarted the job, if it succeeds I still think that we can merge this and keep an eye on it